tcell_driver.go

```diff
@@ -159,7 +159,7 @@ func pollEvent() gocuiEvent {
 			mod = 0
 			ch = rune(0)
 			k = tcell.KeyCtrlSpace
-		} else if mod == tcell.ModCtrl || mod == tcell.ModShift {
+		} else if mod == tcell.ModCtrl || mod == tcell.ModShift && ch != 0 {
```

Comment:
Is spacebar affected by this? I think for spacebar we set `ch=0`, so if it's hit with shift, is it affected or not? Maybe it makes it work for shitf+space? Just curious if you tried it. Thanks

Reply:
I have some logging enabled in my fork that prints the names of the event of tcell (they have a pretty print method) and shift space seems to just be `rune[ ]` whereas they print ctrl space as `Ctrl+Space` so don't think they support shift space but would have to look deeper to be 100% sure

Reply:
```suggestion
		} else if mod == tcell.ModCtrl || mod == tcell.ModShift && (ch != 0 || k == 32) {
```
So maybe in such case it would be better to remove the shift modifier for spacebar like this?

Reply:
Perhaps we should use the contant not hardcode `32`?

Reply:
yeah, makes sense